components/brave_rewards/browser/rewards_service_impl.cc

```diff
@@ -1295,6 +1295,24 @@ void RewardsServiceImpl::OnRecoverWallet(const ledger::type::Result result) {
   }
 }
 
+std::vector<std::string> RewardsServiceImpl::GetExternalWalletProviders() {
+  std::vector<std::string> providers;
+
+  if (IsBitFlyerRegion()) {
+    providers.push_back(ledger::constant::kWalletBitflyer);
+    return providers;
+  }
+
+  providers.push_back(ledger::constant::kWalletUphold);
+
+#if !defined(OS_ANDROID)
```

Comment:
This is not correct, you have an OS_ANDROID guard around the feature, but the feature itself does not have an os guard. OS guards are used for OS-specific code. If gemini is not available on android then we should have a buildflag for gemini that is disabled by default on android and that flag should be used here (and everywhere else that gemini is referenced

Reply:
resolved